tests/test.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- hosts: all
+  become: True
+  roles:
+    - role: ansible-midonet-repo
+      midonet_version: 'midonet-5.2'
+      mem_username: 'calsoft'
+      mem_password: '3adfF3cd'
```

Comment:
DO NOT put valid credentials in here, ever. I had to revoke this password, affecting many people's work schedule. Put a dummy pwd instead.